bindings/java/nz/mega/sdk/DelegateMegaListener.java

```diff
@@ -2,6 +2,25 @@
 
 import java.util.ArrayList;
 
+/**
```

Comment:
Same comment as for `DelegateMegaGlobalListener` class, and all following (which will probably have the same issue).